io.github.dweymouth.supersonic.yml

```diff
@@ -0,0 +1,220 @@
+        - --disable-debug
+        - --disable-doc
+        - --disable-programs
+        - --disable-static
+        - --enable-encoder=png
+        - --enable-gnutls
+        - --enable-gpl
+        - --enable-shared
+        - --enable-version3
+        - --enable-libaom
+        - --enable-libass
+        - --enable-libdav1d
+        - --enable-libfreetype
+        - --enable-libmp3lame
+        - --enable-libopus
+        - --enable-libtheora
+        - --enable-libv4l2
+        - --enable-libvorbis
+        - --enable-libvpx
+      sources:
+        - type: git
+          url: https://github.com/FFmpeg/FFmpeg.git
+          mirror-urls:
+            - https://git.ffmpeg.org/ffmpeg.git
+          commit: ea3d24bbe3c58b171e55fe2151fc7ffaca3ab3d2
+          tag: n6.0
+          x-checker-data:
+            type: git
+            tag-pattern: ^n([\d.]{3,7})$
+
+  # this is used to bundle the final supersonic command correctly and
+  # provides the /app/bin/fyne command to the next module
+  - name: finecmd
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+      # TODO: remove this and get each dependency properly, then add
+      # `GO111MODULE: off` above
+      #
+      # package list could be generated with:
+      # https://github.com/divVerent/aaaaxy/blob/main/scripts/go-vendor-to-flatpak-yml.sh
+      # https://github.com/flathub/com.yktoo.ymuse/blob/7c33add48cfc6788b9ddf0c92ee7213f40c6c78e/com.yktoo.ymuse.yml#L57-L63
+      #
+      # this doesn't work so well:
+      # https://github.com/flatpak/flatpak-builder-tools/tree/master/go-get
+      # ... because it generates git links, which are much slower
+      #
+      # other examples:
+      # https://github.com/flathub/me.kozec.syncthingtk/blob/cf25336132b96c514f29d5a5322874e847aba150/syncthing.yaml
+      #
+      # also used:
+      # https://github.com/search?q=org%3Aflathub+++mpv&type=code
+      build-args:
+        - --share=network
+    build-commands:
+      - . /usr/lib/sdk/golang/enable.sh && go install -v fyne.io/fyne/v2/cmd/fyne@v2.3.3
+
+  - name: supersonic
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+      # TODO: remove this and get each dependency properly, then add
+      # `GO111MODULE: off` above
+      build-args:
+        - --share=network
+    build-commands:
+      - . /usr/lib/sdk/golang/enable.sh && CGO_LDFLAGS=-L/app/lib go build -v .
+      - . /usr/lib/sdk/golang/enable.sh && CGO_LDFLAGS=-L/app/lib /app/bin/fyne package -os linux -name Supersonic -icon ./res/appicon-500.png
+      - tar -C /app --strip-components=1 -x -v -f /run/build/supersonic/Supersonic.tar.xz
+    sources:
+      - type: git
+        url: https://github.com/dweymouth/supersonic/
+        commit: 440dee15ac1bb42fcd899578576098542ffc4a19
+    rename-desktop-file: /app/share/applications/Supersonic.desktop
```

Comment:
that doesn't belong here. See `man flatpak-manifest`

Also the syntax is wrong.